virttest/guest_agent.py

```diff
@@ -725,3 +725,14 @@ def fsthaw(self, check_status=True):
                     raise
             return ret
         return -1
+
+    def file_operate(self, args, cmd):
+        """
+        Send file operations's guest agent command.
+
+        :params args: It includes guest agent command's parameters.
+        :params cmd: It specify guest agent command.
+        :return: true if the response received, False if return NULL
```

Comment:
This description does not match your code

Reply:
@vivianQizhu  I updated description.  please review it again. Thanks.